jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2628,9 +2628,40 @@ Jobs are run concurrently by default, so you must explicitly require any depende
 | `requires`
 | N
 | List
-| A list of jobs that must succeed for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option. However, if all dependencies of a job are filtered, then that job will not be executed either.
+| A list of jobs that must succeed or attain a certain status for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option. However, if all dependencies of a job are filtered, then that job will not be executed either.
 |===
 
+[.table.table-striped]
+[cols=3*, options="header", stripes=even]
+|===
+| Key | Type | Description
+
+| `job_name`
+| Job Name
+| An upstream job that must `succeed` for the job to start.
```

Comment:
I'd rename this to `required job` to be consistent with what customers write in their config. 
"upstream" is a term we use internally and is not mentioned anywhere else in our docs.